app/components/nav-bar/template.hbs

```diff
@@ -0,0 +1,14 @@
+<!-- Navbar -->
+    <nav class="navbar navbar-inverse">
```

Comment:
Are we keeping this as the basic inverted or are we going to have it fit the KUB color scheme?